vscode-mysql/v2/Dockerfile

```diff
@@ -250,7 +250,7 @@ RUN set -ex; \
 	apt-key list > /dev/null
 
 ENV MYSQL_MAJOR 8.0
-ENV MYSQL_VERSION 8.0.23-1debian10
+ENV MYSQL_VERSION 8.0.25-1debian10
```

Comment:
Was this MySQL bump intended to be part of this change?

Reply:
Yes because they [updated the mysql version](https://dev.mysql.com/downloads/mysql/) which caused builds to fail with the previously used MySQL version.